{
  "uid" : "ed9e35a18a688e9d",
  "name" : "User tries to pay a cart with two RPTs that exists already in GPD in invalid state",
  "fullName" : "User pays a payment carts from existing payment position via nodoInviaCarrelloRPT: User tries to pay a cart with two RPTs that exists already in GPD in invalid state",
  "historyId" : "3d26ee416cc3affe0103c3d8164ab3c5",
  "time" : {
    "start" : 1745834176551,
    "stop" : 1745834181930,
    "duration" : 5379
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: There is no error matching the one searched in the field_name: operation_error_code\n",
  "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.3/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.3/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"src/bdd/wisp/steps/steps.py\", line 235, in debt_position_invalid\n    steputils.check_debt_position_invalid_and_sent_ko_receipt(context)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/home/runner/work/pagopa-platform-integration-test/pagopa-platform-integration-test/src/utility/wisp/steps_utils.py\", line 959, in check_debt_position_invalid_and_sent_ko_receipt\n    raise AssertionError('There is no error matching the one searched in the field_name: operation_error_code')\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: There is no error matching the one searched in the field_name: operation_error_code\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.3/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.3/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"src/bdd/wisp/steps/steps.py\", line 235, in debt_position_invalid\n    steputils.check_debt_position_invalid_and_sent_ko_receipt(context)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/home/runner/work/pagopa-platform-integration-test/pagopa-platform-integration-test/src/utility/wisp/steps_utils.py\", line 959, in check_debt_position_invalid_and_sent_ko_receipt\n    raise AssertionError('There is no error matching the one searched in the field_name: operation_error_code')\n",
    "steps" : [ {
      "name" : "Given systems up",
      "time" : {
        "start" : 1745834176552,
        "stop" : 1745834177391,
        "duration" : 839
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Given a cart of RPTs non-multibeneficiary",
      "time" : {
        "start" : 1745834177392,
        "stop" : 1745834177392,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And a single RPT of type BBT with 1 transfers of which none are stamps",
      "time" : {
        "start" : 1745834177392,
        "stop" : 1745834177392,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And a single RPT of type BBT with 1 transfers of which none are stamps",
      "time" : {
        "start" : 1745834177392,
        "stop" : 1745834177392,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And an existing payment position related to first RPT with segregation code equals to 48 and state equals to DRAFT",
      "time" : {
        "start" : 1745834177393,
        "stop" : 1745834177854,
        "duration" : 461
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "311456c12e90d558",
        "name" : "Step: [an existing payment position related to first RPT with segregation code equals to 48 and state equals to DRAFT] - Sent request",
        "source" : "311456c12e90d558.txt",
        "type" : "text/plain",
        "size" : 1256
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "When the user tries to pay a cart of RPTs on EC website",
      "time" : {
        "start" : 1745834177854,
        "stop" : 1745834178471,
        "duration" : 617
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "3970e00d48deac08",
        "name" : "Step: [the user tries to pay a cart of RPTs on EC website] - Sent request",
        "source" : "3970e00d48deac08.txt",
        "type" : "text/plain",
        "size" : 15119
      }, {
        "uid" : "8efe309fab4f3b48",
        "name" : "Step: [the user tries to pay a cart of RPTs on EC website] - Received response",
        "source" : "8efe309fab4f3b48.txt",
        "type" : "text/plain",
        "size" : 409
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "Then conversion to new model fails in wisp-converter",
      "time" : {
        "start" : 1745834178471,
        "stop" : 1745834179291,
        "duration" : 820
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "25f15c644dcbe3ae",
        "name" : "Step: [conversion to new model fails in wisp-converter]: on sessionId: [14d47b1d-96ca-46bd-838f-4241f2379042] - Received response",
        "source" : "25f15c644dcbe3ae.txt",
        "type" : "text/plain",
        "size" : 5807
      }, {
        "uid" : "96791a302f4b5d2c",
        "name" : "Received response",
        "source" : "96791a302f4b5d2c.attach",
        "type" : "text/html",
        "size" : 5677
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "And the KO receipt is sent",
      "time" : {
        "start" : 1745834179291,
        "stop" : 1745834181929,
        "duration" : 2638
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: There is no error matching the one searched in the field_name: operation_error_code\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.3/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.3/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"src/bdd/wisp/steps/steps.py\", line 235, in debt_position_invalid\n    steputils.check_debt_position_invalid_and_sent_ko_receipt(context)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/home/runner/work/pagopa-platform-integration-test/pagopa-platform-integration-test/src/utility/wisp/steps_utils.py\", line 959, in check_debt_position_invalid_and_sent_ko_receipt\n    raise AssertionError('There is no error matching the one searched in the field_name: operation_error_code')\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "dfa42ad0bd59e15d",
        "name" : "Step: [the KO receipt is sent]: on IUV: [923067091478238] - Received response",
        "source" : "dfa42ad0bd59e15d.txt",
        "type" : "text/plain",
        "size" : 55771
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 8,
    "attachmentsCount" : 6,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "tag",
    "value" : "nodo_invia_carrello_rpt"
  }, {
    "name" : "tag",
    "value" : "runnable"
  }, {
    "name" : "tag",
    "value" : "unhappy_path"
  }, {
    "name" : "feature",
    "value" : "User pays a payment carts from existing payment position via nodoInviaCarrelloRPT"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "runnable", "nodo_invia_carrello_rpt", "unhappy_path" ]
  },
  "source" : "ed9e35a18a688e9d.json",
  "parameterValues" : [ ]
}